spec/models/declaration_spec.rb

```diff
@@ -14,4 +14,21 @@
     it { is_expected.to validate_presence_of(:declaration_type) }
     it { is_expected.to validate_presence_of(:declaration_date) }
   end
+
+  describe "scopes" do
+    describe ".billable" do
+      it "returns declarations with billable states" do
+        billable_declarations = Declaration::BILLABLE_STATES.map do |state|
+          create(:declaration, state:)
+        end
+
+        non_billable_declarations = Declaration::NON_BILLABLE_STATES.map do |state|
+          create(:declaration, state:)
+        end
+
+        expect(Declaration.billable).to match_array(billable_declarations)
+        expect(Declaration.billable).not_to include(*non_billable_declarations)
```

Comment:
We shouldn't need this as the `match_array` will ensure it only contains billable declarations.

Reply:
Addressed in https://github.com/DFE-Digital/npq-registration/pull/1435/commits/784877789af410b288ac63f2563e59419f6909bf